### DISCUSSION TITLE
[Feature Request] Allow more than 1000 messages per feed

### DISCUSSION THREAD

**Discussion by dailylhama** [spacecowboy/Feeder#344] [Ideas]
I have a feed that updates quite regularly, i.e. 400-500 messages each month. I have read (or marked as seen) most of them, but the ones that I left unread for later disappear after a while once the message limit is reached.

I propose to either
- allow bigger/unlimited message limits or
- only delete read messages upon reaching the message limit.

Thank you!

**Comment by spacecowboy** [comment]
If want to prevent them from beinf deleted, then use the 'Save Article' feature
